src/link.zig

```diff
@@ -435,6 +435,24 @@ pub const File = struct {
         }
     }
 
+    /// Called from within CodeGen to retrieve the symbol index of a global symbol.
+    /// If no symbol exists yet with this name, a new one will be created instead.
```

Comment:
Could you add a clarification here, that if the symbol didn't yet exist, we create a new undefined global symbol which will hopefully get resolved once all relocatables are re-linked?